sources/api/schnauzer/src/helpers.rs

```diff
@@ -1346,6 +1356,138 @@ handlebars_helper!(any_enabled: |arg: Value| {
     result
 });
 
+/// This helper writes out the default OCI runtime spec.
+///
+/// Default settings are specified in the following files:
+/// * sources/models/shared-defaults/oci-resource-limits.toml
+/// * sources/models/shared-defaults/oci-capabilities.toml
```

Comment:
Move these references to the models file where the structs are declared, so people can see where to get/set defaults there